Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -4381,9 +4440,62 @@ ObservableList<ControlProperty> IWindowExplorer.GetElementProperties(ElementInfo
                 ElementInfo.ElementObject = el;
             }
 
-            //Learn more properties
+            //Base properties 
```

Comment:
else code in row 4437 should call to function which will get the ElementObject instead implementing it in this function (getting the Xpath...).
the all idea is to stop doing the same thing over and over again in multiple functions.
this function should handle only Properties

Reply:
do you mean to find the element by its locators?
what if it comes to this function with no locators?

Reply:
no, just to pull the few code to seperate function, you can do it later with the massive cleanup needed